components/MembersList/requests.jsx

```diff
@@ -11,7 +10,7 @@ export const fetchVeolasBalance = ({ account }) => new Promise((resolve, reject)
     .balanceOf(account)
     .call()
     .then((response) => {
-      resolve(formatToEth(response));
```

Comment:
I removed this because it was adding a decimal place and BigNumber didn't like it. I manually tested the existing check on the Join action and it still works.